guides/k8-akka-cluster/src/main/paradox/index.md

```diff
@@ -0,0 +1,465 @@
+                    <entryPoint>
+                        java -cp '/maven/*' -DactorSystemName=${AKKA_ACTOR_SYSTEM_NAME} -Dakka.actor.provider=cluster -Dakka.remote.netty.tcp.hostname="$AKKA_REMOTING_BIND_HOST" -Dakka.remote.netty.tcp.port="$AKKA_REMOTING_BIND_PORT" -Dakka.cluster.seed-nodes.0="akka.tcp://${AKKA_ACTOR_SYSTEM_NAME}@${AKKA_SEED_NODE_HOST}:${AKKA_SEED_NODE_PORT}" com.mycompany.MainClass
+                    </entryPoint>                    
+                </build>
+            </image>
+        </images>
+    </configuration>
+</plugin>
+```
+
+When building a container image for the application, the base image [openjdk:8-jre-alpine](https://hub.docker.com/_/openjdk/) will be used to provide a working JRE 8 installation for the application to run on. The container image name expression is `%g/%a:%l` where `%g`, `%a`, and `%l` are Maven project's group id, artefact id, and the image tag. When published, our image will tagged with `latest` and `${project.version}`
+
+The fabric8 Maven plugin will place all the jar files under `/maven` directory which need to be added to the JVM's classpath. Also note that we have added the Akka clustering related system properties as part of the JVM options.
+
+Replace `com.mycompany.MainClass` with the actual main class of your application.
+
+Execute the following command to containerize and publish your application's Docker image:
+
+```bash
+mvn clean package docker:build
+```
+
+### Creating Kubernetes Service for Akka remoting
+
+Next we will declare the Akka remoting port we'd like to expose from our Pod using Kubernetes Service so it can be referenced from one Pod to another.
+
+Execute the following command to create Kubernetes Service for the application's Akka remoting port:
+
+```bash
+cat << EOF | kubectl create -f -
+{
+  "apiVersion": "v1",
+  "kind": "Service",
+  "metadata": {
+    "labels": {
+      "app": "myapp"
+    },
+    "name": "myapp-akka-remoting"
+  },
+  "spec": {
+    "clusterIP": "None",
+    "ports": [
+      {
+        "port": 2551,
+        "protocol": "TCP",
+        "targetPort": 2551
+      }
+    ],
+    "selector": {
+      "app": "myapp"
+    }
+  }
+}
+EOF
+```
+
+This Kubernetes Service has `myapp-akka-remoting` as its name. The service exposes TCP port `2551` which is the application's Akka remoting port.
+
+All Pods that have the label `app` with the value of `myapp` will expose the TCP port `2551`.
+
+To view the list of Kubernetes services, use the following command:
+
+```bash
+kubectl get services
+```
+
+We can also filter the Kubernetes services based on our created label `app=myapp`:
+
+```bash
+kubectl get services --selector=app=myapp
+```
+
+### Creating Kubernetes StatefulSet resource
+
+Once we have the application containerized and published, and the Kubernetes Service declared for the application's Akka remoting port, we are ready to deploy the application.
+
+Execute the following command to deploy the application:
```

Comment:
Most likely yes, but I think generally devops or ops person will be in control of this file.

Since this is a guide of how to do Akka cluster, I'm thinking about showing something that they can copy and paste. I suppose I can achieve copy-paste using file as well, but then they would have this file lying around.

Perhaps I can mention about saving this into a file in version control for the actual deployment  - wdyt?